### PR TITLE
Doc 553 new unit tests and tweaks

### PR DESCRIPTION
**Pull request by GodsonLeigh**

### CODE REVIEW

File: test/unit/generator/get-document-definition.test.js [hunk 1]
```diff
@@ -2,41 +2,41 @@ const styles = require('../../../app/generator/styles')
 const getDocumentDefinition = require('../../../app/generator/get-document-definition')
 const mockStatement = require('../../mocks/mock-statement')
 const { A4 } = require('../../../app/generator/page-sizes')
-const { STATEMENT } = require('../../../app/constants/document-types')
+const { STATEMENT, SFI23QUARTERLYSTATEMENT } = require('../../../app/constants/document-types')
 
 describe('get document definition', () => {
   test('includes A4 paper size', () => {
-    const result = getDocumentDefinition(mockStatement, STATEMENT)
+    const result = getDocumentDefinition(mockStatement, STATEMENT, SFI23QUARTERLYSTATEMENT)
```
Comment (AbidemiAdio) [inline test/unit/generator/get-document-definition.test.js:9]: Wrong signature for getDocumentDefinition

File: test/unit/generator/get-document-definition.test.js [hunk 2]
```diff
@@ -2,41 +2,41 @@ const styles = require('../../../app/generator/styles')
...
     expect(result.pageSize).toBe(A4)
   })
 
   test('includes all defined styles', () => {
-    const result = getDocumentDefinition(mockStatement, STATEMENT)
+    const result = getDocumentDefinition(mockStatement, STATEMENT, SFI23QUARTERLYSTATEMENT)
```
Comment (AbidemiAdio) [inline test/unit/generator/get-document-definition.test.js:14]: As above

File: test/unit/generator/get-document-definition.test.js [hunk 3]
```diff
@@ -2,41 +2,41 @@ const styles = require('../../../app/generator/styles')
...
   test('sets default style as default style', () => {
-    const result = getDocumentDefinition(mockStatement, STATEMENT)
+    const result = getDocumentDefinition(mockStatement, STATEMENT, SFI23QUARTERLYSTATEMENT)
     expect(result.defaultStyle).toBe(styles.default)
   })
 
   test('sets left margin', () => {
-    const result = getDocumentDefinition(mockStatement, STATEMENT)
-    expect(result.pageMargins[0]).toBe(42.525)
+    const result = getDocumentDefinition(mockStatement, STATEMENT, SFI23QUARTERLYSTATEMENT)
```
Comment (AbidemiAdio) [inline test/unit/generator/get-document-definition.test.js:24]: As above

File: test/unit/generator/get-document-definition.test.js (hunk 1 above)
Comment (GodsonLeigh) [reply test/unit/generator/get-document-definition.test.js]: I have rewritten the logic for the tests

**Review by AbidemiAdio — APPROVED** [review]
QED

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**